core/markets/storage/provider/impl/provider_impl.cpp

```diff
@@ -681,10 +681,7 @@ namespace fc::markets::storage::provider {
                                 deal_context->deal->is_fast_retrieval});
     FSM_HALT_ON_ERROR(
         maybe_piece_location, "Unable to locate piece", deal_context);
-    FSM_HALT_ON_ERROR(
-        recordPieceInfo(deal_context->deal, maybe_piece_location.value()),
-        "Record piece failed",
-        deal_context);
+    deal_context->maybe_piece_location = maybe_piece_location.value();
     // TODO(a.chernyshov): add piece retry
```

Comment:
I suppose the comment is not relevant anymore.

Reply:
It could be relevant, if sealing cannot store it for now. But after sometime it could